db/schema.rb

```diff
@@ -48,16 +48,6 @@
     t.index ["user_id"], name: "index_api_secrets_on_user_id"
   end
 
-  create_table "article_collections", force: :cascade do |t|
-    t.bigint "article_id"
-    t.datetime "created_at", null: false
-    t.string "status", default: "Unread"
-    t.datetime "updated_at", null: false
-    t.bigint "user_collection_id"
-    t.index ["article_id"], name: "index_article_collections_on_article_id"
-    t.index ["user_collection_id"], name: "index_article_collections_on_user_collection_id"
-  end
-
```

Comment:
i commented on this earlier but i see now it has been taken out

Reply:
Yep erroneous code that was used at some point while getting help. It never got deleted apparently.